incubator/inbucket/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+description: Inbucket is an email testing application
+name: inbucket
+appVersion: 1.2.0
+version: 0.0.1
+keywords:
+  - inbucket
+  - mail
+  - smtp
+  - pop3
+  - email
+  - testing
+  - disposable webmail
+home: https://www.inbucket.org/
+sources:
+  - https://github.com/jhillyerd/inbucket
+maintainers:
+  - name: cpanato
+    email: ctadeu@gmail.com
+  - name: jhillyerd
```

Comment:
E-mail address missing.

Reply:
Can you add an icon?

Reply:
email added. there is no icon until now. I will check with the onwer of the repo 😄